builtin/settingtypes.txt

```diff
@@ -1619,7 +1619,7 @@ mgvalleys_np_cave1 (Cave noise #1) noise_params_3d 0, 12, (61, 61, 61), 52534, 3
 #    Second of two 3D noises that together define tunnels.
 mgvalleys_np_cave2 (Cave noise #2) noise_params_3d 0, 12, (67, 67, 67), 10325, 3, 0.5, 2.0
 
-#    The depth of dirt or other biome filler node.
+#    Variation of biome filler depth.
```

Comment:
The information that biome filler is usually dirt is likely useful.

Reply:
"dirt" seems MC/MTG-specific.

Most other mapgens also have a `mg< name>_np_filler_depth` setting. If this description is changed again, please make sure that all `mg< name>_np_filler_depth` settings still have the same description.